tests/fast-integration/telemetry-test/suite.js

```diff
@@ -409,6 +411,35 @@ describe('Telemetry Operator', function() {
           await k8sDelete(secondPipeline);
         });
       });
+
+
+      context('Filter Processor', function() {
+        const testApp = loadTestData('tracepipeline-test-app.yaml');
```

Comment:
We should try to reduce the dependencies to other Kyma components. Let's make it a regular deployment and not a function. Also the chain of different services that are called seems to be not really necessary for this test (just more things that can break and cause flakyness). Maybe just deploy a simple webserver (nginx image or similar) and expose it as a VirtualService.